packages/web/src/utils/launchpad-get-tier-number.ts

```diff
@@ -8,10 +10,23 @@ export const getTierNumber = (tier: TierType): number => {
 export const getTierDuration = (tier: TierType) => {
   switch (tier) {
     case "TIER30":
-      return "1 Month";
+      return i18n?.t("Launchpad:common.tierDuration.1month");
```

Comment:
In my opinion, it seems better to take over a function that formats i18n as an argument than to use an i18n instance.
It seems like a good idea to have utility functions behave as pure functions.